data/data_80DED44.s

```diff
@@ -223,13 +223,13 @@ gUnknown_80E04B4: @ 80E04B4
 
 	.global gUnknown_80E04F4
 gUnknown_80E04F4: @ 80E04F4
-        .string "#+...Oh~2c wait~2c sorry.#P"
-        .string "#+You already have the \n"
-        .string "#+#C4$h Friend Area#R.#P"
-        .string "#+I~27m sorry that my reward is\n"
-        .string "#+something you already have.#P"
-        .string "#+In its place~2c #CN$m0#R~2c\n"
-        .string "#+you can have #CG1~2c000#R {POKE}.\0"
+        .string "{CENTER_ALIGN}...Oh{COMMA} wait{COMMA} sorry.{EXTRA_MSG}"
+        .string "{CENTER_ALIGN}You already have the \n"
+        .string "{CENTER_ALIGN}{COLOR_1 GREEN}$h Friend Area{END_COLOR_TEXT_1}.{EXTRA_MSG}"
```

Comment:
Looks like `$h` is an argument for friend area name if you want to make a macro for that. There are a few other places where this occurs.

Reply:
Defined `$h` as `{ARG_FRIEND_AREA}`